config/config.js

```diff
@@ -88,6 +88,21 @@ var validateSecureMode = function (config) {
 };
 
 /**
+ * Validate Session Secret parameter is not set to default in production
```

Comment:
Nice, I like this addition.
Can you update the message to be a bit more verbose in telling the user where to change it? i.e `config/env/production.js`